src/config/config.js

```diff
@@ -56,6 +57,9 @@ function getDefaultConfig() {
    *     maxSamplesPerBulkRequest: [INTEGER],
    *     sampleUpsertQueueTime: [INTEGER],
    *
+   *     // optional proxy to be used for Refocus
+   *     proxy: "[REFOCUS_PROXY]",
```

Comment:
move this up to line 48 so it's grouped along with the other data getting passed in from command-line arg or env var?